package.nls.json

```diff
@@ -155,6 +180,7 @@
     "AWS.command.submitFeedback": "Submit Quick Feedback...",
     "AWS.command.downloadSchemaItemCode": "Download Code Bindings",
     "AWS.command.viewLogs": "View AWS Toolkit Logs",
+    "AWS.command.viewLogs.cn": "View Amazon Toolkit Logs",
```

Comment:
we could rename the command to `View Logs`. Would be very cumbersome for e.g. documentation to have different command names for different regions.